src/mbgl/renderer/tile_pyramid.hpp

```diff
@@ -24,7 +24,7 @@ class RenderTile;
 class RenderStyle;
 class RenderedQueryOptions;
 class SourceQueryOptions;
-class TileParameters;
+struct TileParameters;
```

Comment:
We already forward declare some `struct` objects: https://github.com/mapbox/mapbox-gl-native/blame/master/include/mbgl/util/interpolate.hpp#L19 https://github.com/mapbox/mapbox-gl-native/blob/master/src/mbgl/gl/attribute.hpp#L297

Are you suggesting we should enforce `class` for all cases? I'd be :+1: to that, just want to make sure we apply the same behavior everywhere.